### ISSUE TITLE
Increased selenium level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5952]
**Preferred term label:** Increased selenium level

**Synonyms**

**Definition (free text, please give PubMed ID)** An increased concentration of selenium in the blood., PMID:25519493

**Parent term (use hpo.jax.org/app)** Abnormal circulating selenium concentration HP:0031903

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP_0033191
